tests/net_test.go

```diff
@@ -3,11 +3,33 @@ package fargo_test
 // MIT Licensed (see README.md) - Copyright (c) 2013 Hudl <@Hudl>
 
 import (
+	"fmt"
+	"net/http"
+	"testing"
+
 	"github.com/hudl/fargo"
 	. "github.com/smartystreets/goconvey/convey"
-	"testing"
 )
 
+func shouldNotBearAnHTTPStatusCode(actual interface{}, expected ...interface{}) string {
+	if code, present := fargo.HTTPResponseStatusCode(actual.(error)); present {
+		return fmt.Sprintf("Expected:	no HTTP status code\nActual: 		%d", code)
```

Comment:
I misinterpreted what I saw [here in the assertion failure messages used by GoConvey](https://github.com/smartystreets/assertions/blob/master/messages.go), which I was trying to imitate. I'll fix it, aligning the fields with spaces instead.